docs/Migration_guide_from_v4_to_v5.md

```diff
@@ -71,6 +71,7 @@ framework | removed | n/a
 rescue_rake_exceptions | removed | n/a
 user_attributes | removed | n/a
 test_mode | removed | n/a
+ignore_only | removed | n/a
```

Comment:
Could you move this below `ignore_user_agent` (line 60) and instead of `removed` mention `[add_filter](#filtering)`, just like other ignore options do?